### PR TITLE
Reference cleanup

### LINKED ISSUE #11: Convert URL citations to hyperlinks
Follow the criteria for [citeable work](https://github.com/greenelab/meta-review/pull/9#discussion_r142155007) in #9

### PR DESCRIPTION
**Pull request by agitter**
I converted URL citations to links as appropriate and manually fixed the CSL for the remaining URL citations.  I also cleaned up a few other things, like our duplicate back matter files.

Closes #11.

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -98,13 +99,13 @@ The entire process can be orchestrated through GitHub with a web browser if a co
...
+Because our writing process, like others backed by the open git version control system (including Overleaf and Authorea), tracks the complete commit history, it also enables detailed retrospective contribution analysis.
 `TODO: confirm Overleaf and Authorea provide this type of git integration versus something more coarse`
 
 ### Authorship
 
-To determine authorship we followed the International Committee of Medical Journal Editors (ICMJE) guidelines and used GitHub to track contributions.
-ICMJE recommends authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript [@url:http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html].
+To determine authorship we followed the [International Committee of Medical Journal Editors (ICMJE) guidelines](http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html) and used GitHub to track contributions.
```
Comment (dhimmel) [inline content/02.main-text.md]: maybe just hyperlink `guidelines` as hyperlinking the entire thing will be visually prominent for a hyperlink of minimal importance.

**Review by dhimmel — APPROVED** [review]

File: content/02.main-text.md (hunk 1 above)
Comment (agitter) [reply content/02.main-text.md]: 👍

**Comment by agitter** [comment]
I'm going to build the manuscript locally to check the manually-adjusted references before merging.

@dhimmel It's taking a while to build my initial cache.  Is that expected or a problem with my machine?

**Comment by agitter** [comment]
The manual references look good overall in my local build.  Two minor issues I noticed are:

1. bioRxiv papers show up with _Cold Spring Harbor Laboratory_ as the CSL `venue`.
image
This might be out of our control.  The JSON file from CrossRef has `"container-title":[]`.  I was curious why my reference manager (Zotero) showed bioRxiv as the journal when I imported the preprint.  It looks like Zotero uses the Highwire Press metatags embedded in the HTML, which show `<meta name="citation_journal_title" content="bioRxiv" />`.

2. References without authors are missing whitespace
image

Those aren't specific to this pull request and could be addressed in greenelab/manubot-rootstock#53 or elsewhere.

**Pull request merged**